src/components/DayPicker.jsx

```diff
@@ -37,6 +37,7 @@ const MONTH_PADDING = 23;
 const DAY_PICKER_PADDING = 9;
 const PREV_TRANSITION = 'prev';
 const NEXT_TRANSITION = 'next';
+const RTL_TRANSLATION_VALUE = -295;
```

Comment:
hmm i was sure i changed that magic number. anyway it should get the negative size of `getCalendarMonthWidth()`.